app/components/nav.tsx

```diff
@@ -1,29 +1,43 @@
 import React from 'react';
 import Link from 'next/link';
+import { PageHeader, LogoContainer } from '@lib';
+import NasaLogo from '../../public/images/nasa-logo';
 
-interface NavItem {
-  // @TODO: To be replaced with exposed NavItem type from veda-ui
-  path: string;
-  title: string;
-  subItems?: NavItem[];
-}
-
-const navItems: NavItem[] = [
+const navItems = [
+  // @TODO: This should use the NavLinkType from veda-ui...
```

Comment:
is this still to do?

Reply:
Yes!! this TODO i'm actually working on actively with exposing some of these interfaces from veda-ui. It's meant to stay